docs/source/_installation/mac_silicon.rst

```diff
@@ -0,0 +1,104 @@
+   cd MDSuite
+
+This should download the MDSuite code and change you into the main directly where you will see the file structure.
+
+TensorFlow Install
+^^^^^^^^^^^^^^^^^^
+**The following is taken from issue #153 frpm the apple/tensorflow_macos repository found**
+`here <https://github.com/apple/tensorflow_macos/issues/153>`_
+
+In order to satisfy all of the dependencies we will need to use MiniForge to create a virtual environment.
+
+1.) Install MiniForge
+
+    Go `here <https://github.com/conda-forge/miniforge#miniforge3>`_ and install MiniForge for your machine. Once it has
+    been installed you should run some checks to make sure your commands are pointing to the correct executables.
+
+    .. code-block:: bash
+
+       which python  # should point to ~/miniforge/bin/python.
+       which pip  # should point to a miniforge directory.
+
+    If this is not the case, you will need to look into adding these paths to your ~/.zshrc or ~/.bashrc or whichever
+    rc file you rae using.
+
+2.) Download :download:`environment.yml <../_static/environment.yml>`
+
+3.) Go to your terminal and run the following commands from the directory in which you have the environment.yml file:
+
+    .. code-block:: bash
+
+       conda env create --file=environment.yml --name=tf_macos
+       conda activate tf_macos
+       pip install --upgrade --force --no-dependencies https://github.com/apple/tensorflow_macos/releases/download/v0.1alpha2/tensorflow_addons_macos-0.1a2-cp38-cp38-macosx_11_0_arm64.whl https://github.com/apple/tensorflow_macos/releases/download/v0.1alpha2/tensorflow_macos-0.1a2-cp38-cp38-macosx_11_0_arm64.whl
+
+4.) Go and test your installation by running:
+
+    .. code-block:: bash
+
+       python
+       >>> import tensorflow as tf
+
+    If this does not fail, the installation worked!
+
+Dependencies Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^
+This is where the installation can get a little bit difficult. Several MDSuite packages will require conda installation
+rather than pip due to their use of a FORTRAN compiler. This can be achieved by running the following:
+
+.. code-block:: bash
+
+   conda install numpy
+   conda install scipy
+   conda install pandas
+   conda install matplotlib
+   pip install h5py
+
+This should take care of the difficult installations. There is still one more step to the installation just to make sure
+nothing goes wrong when we run pip install on the MDSuite directory.
+
+Updating requirements.txt
+^^^^^^^^^^^^^^^^^^^^^^^^^
+We need to remove some of the dependencies from the requirements.txt file in order to make sure pip doesn't try to
+uninstall and reinstall packages that require conda. Open the requirements.txt file in the main directory of MDSuite
+and remove the following packages from it:
+
+* tensorflow
+* h5py
+* numpy
+* scipy
+* pandas
+* matplotlib
+
+Once this is done, run the following to complete the install:
+
+.. code-block:: bash
+
+   pip install -e .
```

Comment:
I don't think that `-e` should be the default?

Reply:
Yeah I was unsure about this. I made the call that if they use the mac M1 they might need to change code around but that's probably not the right way to think about it. I can remove it.